packages/spaces/commands/vpn/config.js

```diff
@@ -3,20 +3,20 @@
 const cli = require('heroku-cli-util')
 const co = require('co')
 
-function displayVPNConfigInfo (space, config) {
-  cli.styledHeader(`${space} VPNs`)
-  config.ipsec_tunnels.forEach((val, i) => {
+function displayVPNConfigInfo (space, name, config) {
+  cli.styledHeader(`${name} VPNs`)
```

Comment:
I'm thinking this header may not make sense now. Maybe it should read `${name} VPN Tunnels`? Or will this be listing all VPN connections as well?